### ISSUE TITLE
Version 1.9.0 introduces TypeScript issues when building using the vue cli

### ISSUE THREAD

**Issue by shandrolis** [quasarframework/quasar#6360]
If you try to build a project that was created with the Vue-cli (using the quasar plugin) with the new quasar **1.9.0** version, the build will fail with errors. Reverting to an older version of quasar gets rid of these errors.

```
ERROR in <project-root>/node_modules/quasar/dist/types/configuration/electron-conf.d.ts(21,8):
21:8 Cannot find module 'electron-packager'.
```

```
ERROR in <project-root>/node_modules/quasar/dist/types/globals.d.ts(4,26):
4:26 Cannot find module 'cordova'.

```

The above two errors disappear when adding a shims.d.ts file declaring these in our /src.

However, when building the project on our TeamCity agents, we get two additional errors, namely:

```
ERROR in /opt/buildagent/work/3b861d5a47df892a/<project-root>/node_modules/quasar/dist/types/configuration/build.d.ts(102,33):
[16:32:10]    [Step 2/7] 102:33 Property 'devtool' does not exist on type 'WebpackConfiguration'.
```
```
ERROR in /opt/buildagent/work/3b861d5a47df892a/<project-root>/node_modules/webpack-chain/types/index.d.ts(3,24):
[16:32:10]    [Step 2/7] 3:24 Cannot find module 'https'
```

Adding the above two to a shims file does not fix them however.
We are unsure how webpack-chain is related to this whole thing, but hoping you might know. :D

Our building agents are running npm version 6.13.7 (which is the newest available)

**Comment by IlCallo** [comment]
We're on it, thanks for the issue

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.9.2
